rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/notifications/OpenUnityProjectAsFolderNotification.kt

```diff
@@ -76,19 +75,6 @@ class OpenUnityProjectAsFolderNotification(private val project: Project, private
                 })
             }
 
-            val pluginPath: VirtualFile? = baseDir.findFileByRelativePath("Assets/Plugins/Editor/JetBrains/JetBrains.Rider.Unity.Editor.Plugin.Repacked.dll")
-            if (pluginPath == null || !pluginPath.exists()) {
-                notification.setContent(notification.content + "\r\n" +
-                        "Install Rider’s Unity Editor plugin to automatically configure Rider as the default external editor and enable advanced functionality."
-                )
-                notification.addAction(object : InstallEditorPluginAction() {
-                    override fun actionPerformed(e: AnActionEvent) {
-                        super.actionPerformed(e)
-                        notification.hideBalloon()
-                    }
-                })
-            }
-
```

Comment:
Why is this removed?

Reply:
Because we are now installing it if we detect UnityFolderStructure
https://github.com/JetBrains/resharper-unity/pull/806/files#diff-2bca04de557ea96772f6862ec6550ab4R72